src/site/stages/build/drupal/graphql/GetRemainingContent.graphql.js

```diff
@@ -0,0 +1,51 @@
+const sidebarQuery = require('./navigation-fragments/sidebar.nav.graphql');
+const alertsQuery = require('./alerts.graphql');
+const facilitySidebarQuery = require('./navigation-fragments/facilitySidebar.nav.graphql');
+const outreachSidebarQuery = require('./navigation-fragments/outreachSidebar.nav.graphql');
+const icsFileQuery = require('./file-fragments/ics.file.graphql');
+const outreachAssetsQuery = require('./file-fragments/outreachAssets.graphql');
+const homePageQuery = require('./homePage.graphql');
+const allSideNavMachineNamesQuery = require('./navigation-fragments/allSideNavMachineNames.nav.graphql');
+const menuLinksQuery = require('./navigation-fragments/menuLinks.nav.graphql');
+
+// Get current feature flags
+const { cmsFeatureFlags } = global;
+
+// String Helpers
+const {
+  updateQueryString,
+  queryParamToBeChanged,
+} = require('./../../../../utilities/stringHelpers');
+
+/**
+ * Queries for all of the pages out of Drupal
+ * To execute, run this query at http://staging.va.agile6.com/graphql/explorer.
+ */
+module.exports = `
+  query GetAllPages( $onlyPublishedContent: Boolean!) {
```

Comment:
How closely does this resemble the original `GetAllPages`?

It might make sense to refactor `GetAllPages` to use both this and a `NodeQuery` (or something) so we don't unwittingly lose feature parity between the builds.

Given the timing, the risk seems low, but...y'know, just in case.

Also...prolly could use a rename. :wink:

Reply:
I've finished the refactor, and I think it should be easy to simplify once we've fully implemented the tome-sync stuff.